### PR TITLE
Feature editable.measures

### PR DESCRIPTION
**Pull request by kafnevod**

### CODE REVIEW

File: addon/maptools/rectangletool.js [hunk 1]
```diff
@@ -3,31 +3,35 @@ import MapTool from 'ember-flexberry-gis/maptools/maptool';
 export default MapTool.extend({
 
   _layer: null,
+  _measure: null,
 
-  rectangleStarted({ layer }) {
```
Comment (kuzkok) [inline addon/maptools/rectangletool.js]: this code was used in other tools - zoom, zoom out, and identify. You should restore it.

Comment (kafnevod) [inline addon/maptools/rectangletool.js]: What functionality supports this function?
Should I support this finctionality in other measurements scripts:
circletool.js, markertool.ls, polyline.js, polygon.js?
Or i should remain primary code in rectangletool.js and support measurement function in other scritps: rectangeMeasureTool.js, markerMeasureTool.js, circleMeasureTool.js etc...?

Comment (kuzkok) [inline addon/maptools/rectangletool.js]: It was continues draw rectangle. http://jsbin.com/wilake/9/edit?html,js,output

Comment (kafnevod) [inline addon/maptools/rectangletool.js]: Why zoom, zoom out, identify are subclass of  rectangletool? What properties and method (except for rectangleStarted) they inherit?
I think we should discuss this and related (use disableEdit()) questions on Monday...

**Pull request closed without merging**